src/libs/CardUtils.ts

```diff
@@ -69,6 +69,29 @@ function isCorporateCard(cardID: number) {
     return !!allCards[cardID];
 }
 
+/**
+ * @param cardID
```

Comment:
if there is no extra description to @param then please remove it.

This code used to be JS only, so IMO all the empty `@param XXX` are leftovers from JS.